pegasus/test/test_site.rb

```diff
@@ -59,8 +59,9 @@ def test_new_relic_transactions
     ::NewRelic::Agent.expects(:set_transaction_name).with('/')
     assert_equal 200, get('/').status
 
+    # Ensure that code.org/learn redirects to hourofcode.com/learn.
     ::NewRelic::Agent.expects(:set_transaction_name).with('/learn')
-    assert_equal 200, get('/learn').status
+    assert_equal 301, get('/learn').status
```

Comment:
@TurnerRiley just updated this — the unit test is passing, but can you take another look just in case?

Cc @davidsbailey can you take a look at this too and let me know if I need to do anything else unit test wise?